File: week_2_arrays/scores.c
#include <stdio.h>

const int n = 3; // number of test scores

float average(int array[]);

int main(void) {

    int scores[n];

    for(int count = 0; count < n; count++) {
        printf("Enter Score %i: ", count + 1);
        scanf("%i", &scores[count]);
    }

    printf("Average: %f\n", average(scores));

    return 0;
}

float average(int array[]) {

    int sum = 0;

    for(int count = 0; count < n; count++) {
        sum += array[count];
    }

    return (sum / 3.0);
}

File: week_6_python/scores.py
from cs50 import get_int

scores = []

n = get_int("How many scores: ")

for i in range(n):
    score = get_int("Score: ")
    scores.append(score)
    # scores = scores + [score]


average = sum(scores) / len(scores)   
print(f"Average: {average}")